source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -200,7 +209,12 @@ namespace PlayFab
                                 if (obj.first.length() != 0 && obj.second.length() != 0) // no empty keys or values in headers
                                 {
                                     std::string header = obj.first + ": " + obj.second;
-                                    WinHttpAddRequestHeaders(hRequest, std::wstring(header.begin(), header.end()).c_str(), -1, 0);
+                                    if(TryAddHeader(hRequest, std::wstring(header.begin(), header.end()).c_str(), -1, 0))
+                                    {
+                                        SetErrorInfo(reqContainer, "Error in WinHttpAddRequestHeaders attempting to add parameters" + std::to_string(GetLastError()));
```

Comment:
nit: this error message won't have any space between it and the hresult value

it also should probably mention that the value being reported is an HRSEULT